crates/symbolicator/src/services/objects/mod.rs

```diff
@@ -311,12 +303,10 @@ impl ObjectsActor {
                 };
                 meta_cache
                     .compute_memoized(request)
-                    .bind_hub(sentry::Hub::new_from_top(sentry::Hub::current()))
                     .await
                     .map_err(|error| CacheLookupError { file_source, error })
-            };
-            queries.push(query);
-        }
+            }
+        });
```

Comment:
i think the hub binding was kind of done in the wrong place here, should have been on the entire `async move { }` block